netlogo-gui/src/test/window/CompilerManagerTests.scala

```diff
@@ -245,3 +334,23 @@ class DummyIGWidget(source: String, var value: AnyRef = Double.box(0)) extends D
   }
   def valueObject: AnyRef = value
 }
+
+class DummyTeSliderWidget(source: String, var value: AnyRef = Double.box(0)) extends DummyWidget(source) with InterfaceGlobalWidget {
+//  override def classDisplayName =  I18N.gui.get("tabs.run.widgets.slider")
+  def name: String = "te"
+  def updateConstraints(): Unit = { }
+  def valueObject(x: AnyRef): Unit = x match {
+    case a: AnyRef => value = a
+  }
+  def valueObject: AnyRef = value
+  var increment = 1.0
+  var maximum = 100.0
+  var minimum = 0.0
+
+  // this sets the value in the current constraint and then ensures
+  // cached values are updated -- CLB
```

Comment:
Make sure to add a date (at least month/year) for comments.

Reply:
Double-checking, this might be a comment from a PR, in which case you can delete it if not needed.

Reply:
Craig L. Brozefsky

Reply:
Yea, it was copied straight from DummySliderWidget.